### PR TITLE
Wbs 14 create front page

### PR DESCRIPTION
**Pull request by michals96**
image

This is how front-page looks like. Logging/registering functionality is not yet provided as it will be with `WBS-15` subtask. It might not be not the most beautiful page, but my priority is creating whole required functionalities of the service. Increasing it's appearance is something that I would like to jump into whilst having the most important parts implemented. I have also created a jira story & subtask, which are currently in the backlog, for front-end layer improvements as well as for any other less-prioritized improvements.

### CODE REVIEW

**Comment by gachgp** [comment]
Usually people try to avoid commits like 'fix'. If there is a commit with invalid code it is commonly assumed to be not useful in this form, so it is modified with `ammend` or `fix-up`. This combines commits with faults and `fix` commits.

File: Web-banking-app.tex [hunk 1]
```diff
@@ -27,24 +27,23 @@
...
 %% ------------------------ TYTUL PRACY --------------------------------------
-{\bf \huge \textsf{A long not fitting
-into one line title
-of the master thesis}}\\
+{\bf \huge \textsf{Web application developement using WebToolkit
+C++ on the example of a banking service}}\\
 \vspace{14ex}
 %% ------------------------ OPIEKUN PRACY ------------------------------------
-{\Large \sf Supervisor: {\bf \textsf{Prof. Dr. or Doktor}}}\\
+{\Large \sf Supervisor: {\bf \textsf{Dr Eng Grzegorz Gach}}}\\
```
Comment (gachgp) [inline Web-banking-app.tex:44]: Please, use only `Dr`.

File: include/User.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <Wt/WDateTime.h>
+#include <Wt/Dbo/Types.h>
+#include <Wt/Dbo/WtSqlTraits.h>
+#include <Wt/Auth/Dbo/AuthInfo.h>
+
+using namespace Wt;
```
Comment (gachgp) [inline include/User.h:8]: `using namespace` in the header file.

File: include/WebBankingApplication.h [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#ifndef WEBBANKINGAPPLICATION_H_
+#define WEBBANKINGAPPLICATION_H_
```
Comment (gachgp) [inline include/WebBankingApplication.h:2]: Some files are protected with `#ifndef` and some with `#pragma`.

File: include/WebBankingApplication.h [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+#include <Wt/WApplication.h>
+#include <Wt/WContainerWidget.h>
+#include "Session.h"
+
+class Session;
```
Comment (gachgp) [inline include/WebBankingApplication.h:8]: Nice forward declaration.

File: src/HelloApplication.cpp [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+#include <Wt/WContainerWidget.h>
+#include <Wt/WLineEdit.h>
+#include <Wt/WPushButton.h>
+#include <Wt/WText.h>
+
+HelloApplication::HelloApplication(const Wt::WEnvironment& env)
+    : WApplication(env)
+{
+
+    root()->addWidget(Wt::cpp14::make_unique<Wt::WText>("Your name, please ? "));
```
Comment (gachgp) [inline src/HelloApplication.cpp:13]: Why `make_unique` is from `wt` namespace and not `std`?

**Comment by gachgp** [comment]
This one pull request combines modifications in source files `*.h` and `*.cpp` with web page files like `*.css` or `*.gif`. In total over 300 files are modified. It makes it very difficult to perform proper review.

File: Web-banking-app.tex (hunk 1 above)
Comment (michals96) [reply Web-banking-app.tex:44]: - [x] Fixed

File: include/User.h (hunk 2 above)
Comment (michals96) [reply include/User.h:8]: - [x] Fixed

File: include/WebBankingApplication.h (hunk 3 above)
Comment (michals96) [reply include/WebBankingApplication.h:2]: - [x] Fixed

File: src/HelloApplication.cpp (hunk 5 above)
Comment (michals96) [reply src/HelloApplication.cpp:13]: - [x] Fixed

**Pull request merged**